great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py

```diff
@@ -55,7 +55,7 @@ class ExpectMulticolumnSumToEqual(MulticolumnMapExpectation):
         Exact fields vary depending on the values passed to result_format, include_config, catch_exceptions, and meta.
     """
 
-    sum_total: int
+    sum_total: float
```

Comment:
Actually a float per the JSON tests - this sneakily was truncating values and causing some gnarly test failures